api/indexer/indexer.go

```diff
@@ -87,28 +91,28 @@ func (i *Indexer) initBlocks() error {
 		}
 
 		i.blockIDToHeight.Put(blk.Block.GetID(), blk.Block.Hght)
-		i.blockHeightToBlock.Put(blk.Block.Hght, blk)
+		if _, removedBlk := i.blockHeightToBlock.Put(blk.Block.Hght, blk); removedBlk != nil {
+			// the following should not happen, as we maintain only blockWindow entries; however,
+			// if it does, the following would correct that.
```

Comment:
Since the block window can change on restart, this comment is not accurate ie. we may hit this when the block window shortens such that we should remove blocks